### DISCUSSION TITLE
it doesn't work on huawei soundx

### DISCUSSION THREAD

**Discussion by CamelSmoker** [eizedev/AirConnect-Synology#65] [Q&A]
Could anyone help...

**Comment by eizedev** [comment]
Could you please tell me a litte bit more what are you trying to do? 

Which NAS model and DSM Version?  
Which AirConnect-Synology version?
Any errors in the log file? 

Please also [readme](https://github.com/eizedev/AirConnect-Synology/blob/master/README.md) for more information.
